lib/config/definitions.js

```diff
@@ -1040,6 +1040,18 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'kubernetes',
+    description:
+      'Configuration object for Kubernetes renovation. Also inherits settings from `docker` object.',
+    stage: 'package',
+    type: 'json',
+    default: {
+      fileMatch: ['(^|/)[^/]*\\.yaml$'],
```

Comment:
It can be any yaml file? This might result in a large number of files being found/downloaded/checked. Wondering if we need instead for users to opt in to define directory paths or exact file names instead?

Reply:
I put in a check if the file is actually a Kubernetes manifest: https://github.com/renovatebot/renovate/pull/2341/files#diff-7583452aa9c5f13cf45aa2dd8dfb11d4R12

Reply:
The main problem is hitting API limits. If we “accidentally” pull down dozens or hundreds of files then we risk hitting the org/user API limit of 5000 per hour. If there’s no reliable convention for file or directory naming then I think we might start with an empty fileMatch and let users define one to opt in to the functionality. If one day that changes we can always add to it.

Reply:
That would work for me.